beta-src/src/components/ui/WDInfoPanel.tsx

```diff
@@ -23,26 +23,40 @@ const WDInfoPanel: React.FC<WDInfoPanelProps> = function ({
   const [voteState, setVoteState] = React.useState(userCountry.votes);
 
   const toggleVote = (voteName: Vote) => {
+    const voteKey = Vote[voteName];
     const newVoteState = {
       ...voteState,
-      [voteName]: !voteState[voteName],
+      [voteKey]: !voteState[voteKey],
     };
 
     setVoteState(newVoteState);
   };
 
+  const mobileLandscapeLayout =
+    device === Device.MOBILE_LANDSCAPE || device === Device.MOBILE_LG_LANDSCAPE;
+
+  const padding = mobileLandscapeLayout ? "0 6px" : "0 16px";
+
   return (
     <Box>
-      <WDVoteButtons voteState={voteState} toggleVote={toggleVote} />
-      <WDCountryTable
-        maxDelays={maxDelays}
-        /**
-         * always show current user at the top
-         *
-         */
-        countries={[{ ...userCountry, votes: voteState }, ...countries]}
-        device={device}
-      />
+      <Box sx={{ p: padding }}>
+        <WDVoteButtons voteState={voteState} toggleVote={toggleVote} />
+      </Box>
+      <Box
+        sx={{
+          m: "20px 5px 10px 0",
+        }}
+      >
+        <WDCountryTable
+          maxDelays={maxDelays}
+          /**
+           * always show current user at the top
+           *
+           */
+          countries={[{ ...userCountry, votes: voteState }, ...countries]}
+          device={device}
```

Comment:
abc your props always

Reply:
will reorganize the attributes. sorry i missed these